lib/endpoints/ECatMeeting.d.ts

```diff
@@ -0,0 +1,119 @@
+    /**
+     * Get info on a meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method get
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @param [opts.occurrenceId] ID for the meeting occurrence
+     * @param [opts.showAllOccurrences=false] if truthy,
+     *   retrieves all past occurrences
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meeting#responses}
+     */
+    get(opts: {
+        meetingId: number;
+        occurrenceId?: string;
+        showAllOccurrences?: boolean;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Create a new meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method create
+     * @param opts object containing all arguments
+     * @param opts.userId the user ID or email address of the user
+     * @param opts.meetingObj Zoom meeting object with meeting details {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    create(opts: {
+        userId: string;
+        meetingObj: ZoomMeeting;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Update a meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method update
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @param opts.meetingObj Zoom meeting object with updated details {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingupdate#request-body}
+     * @param [opts.occurrenceId] ID for the meeting occurrence
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    update(opts: {
+        meetingId: number;
+        meetingObj: ZoomMeeting;
+        occurrenceId?: string;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Delete a meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method delete
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @param [opts.occurrenceId] ID for the meeting occurrence
+     * @param [opts.notifyHosts=false] if truthy,
+     *   sends cancellation email to hosts
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    delete(opts: {
+        meetingId: number;
+        occurrenceId?: string;
+        notifyHosts?: boolean;
+    }): Promise<ZoomMeeting>;
```

Comment:
Oooh super subtle question. Sometimes, a meeting has lots of recurrences and instead of deleting the whole series, we just want to delete one occurrence (example: you have a meeting that recurs every Thursday and we just want to delete the occurrence that is happening on Thanksgiving)

Reply:
Do the occurrences for a ZoomMeeting include future scheduled AND past events?

Reply:
I _think_ just past occurrences

Reply:
That's Ok, I was curious